jalapeno-helm/templates/processors/telegraf-egress/processors-telegraf-egress-cm.yaml

```diff
@@ -27,12 +27,13 @@ data:
       database = "{{ (index .Values.influxdb.influxdb.extraEnvVars 0).value }}"
       username = "{{ (index .Values.influxdb.influxdb.extraEnvVars 1).value }}"
       password = "{{ (index .Values.influxdb.influxdb.extraEnvVars 2).value }}"
-      precision = "s"
       timeout = "5s"
 
     [[outputs.file]]
-      files = ["metrics.out"]
+      # Telegraf 1.29 crashes as it can not write to "metrics.out". Following default outputs.file configuration:
+      # https://github.com/influxdata/telegraf/blob/ee18866f3552bbc1debf25ce4e814efbb607d957/plugins/outputs/file/README.md
```

Comment:
Please move these comments from the config map to the PR description if not already there.